front/src/components/QrReader/QrReaderContainer.tsx

```diff
@@ -37,6 +41,31 @@ function QrReaderContainer({ onSuccess }: IQrReaderContainerProps) {
     [setIsProcessingQrCode],
   );
 
+  // TODO: copy, prettier
+  const checkCameraPermission = () => {
+    // TODO: mock this for tests
+    navigator.mediaDevices
+    .getUserMedia({
+      audio: false,
+      video: true,
+    })
+    .catch((error) => {
+      if (error.name === "NotAllowedError") {
+        // eslint-disable-next-line no-console
+        console.error(
+          "User needs to grant this page permission to access the camera.",
+        );
+        setIsCameraNotPermited(true);
+      } else {
+        // eslint-disable-next-line no-console
+        console.error(`getUserMedia error: ${error.name}`, error);
```

Comment:
the triggerError function also includes a console.error. So, I would rather like you to use that.